statusbar: improve formatting

eszkadev opened the pull request:
statusbar: use ui-badge for items
    
    Statusbar items which are not active previously were grey.
    Recover that styling and do cleanup to not specify CSS in the
    JS code.
    
------------------------------------------

    statusbar: better strings formatting
    
After:
![status-badge](https://github.com/CollaboraOnline/online/assets/5307369/ea8d8bb7-1f5b-4cc8-9985-f55c2e61379b)

pedropintosilva reviewed (changes requested):
Awesome Szymon, Thanks!

A note, would be nice to don't style them all as disabled. What I mean is that some of those are not status but navigational labels. Example: Sheet 1 of 1 is not a status label so doesn't have enable/disabled or on/off status so it should be just text without background (we can now or later simple change the text color to somehting like var(--color-text-lighter)). PermissionMode also should not be treated as an off/on (there is Edit and Read-only mode but it's not really perceived as boolean for the user and rather just different modes)

eszkadev commented:
Ok, fixed. I misunderstood. I thought it is only for visual grouping of words for single item :)
I removed `!important` where I can for easier modification later. Now it takes style from `disabled` rules.

Now it is like in 23.05:
![status-badge2](https://github.com/CollaboraOnline/online/assets/5307369/de8e7fb2-a77e-4079-bf83-6c0535ba2c2e)
